pycalphad/tests/test_utils.py

```diff
@@ -0,0 +1,23 @@
+"""
+The utils test module contains tests for pycalphad utilities.
+"""
+
+from pycalphad import Database
+from pycalphad.core.utils import filter_phases
+
+from pycalphad.tests.datasets import ALNIPT_TDB
+
+ALNIPT_DBF = Database(ALNIPT_TDB)
+
```

Comment:
Needs an integration test to show the equilibrium logic is working. Ideally for both ConditionErrors